### DISCUSSION TITLE
How to have the worked hours and the sold hours

### DISCUSSION THREAD

**Discussion by bantunesm** [kimai/kimai#2522] [Questions & Answers]
Hello guys,

First of all, thank you Kevin for this great app, it's easy to install & use. It goes straight to the point ! I tried the csv import by command line and it worked like a charm 🤌

My question:
How can I dissociate sold hours and worked hours ?
Let's say I have a project of 30 hours and I sell it 1500 $ (30 x 50$).
By the end of the project I spent 25 hours but the invoice needs to be calculated on the initial scope of 30 hours.

**Comment by kevinpapst** [comment]
Kimai calculates what was recorded. So you can either create a placeholder entry for 5 hours or you buy the [expenses plugin](https://www.kimai.org/store/expenses-bundle.html), which allows to enter arbitrary amounts that will be charged on your invoices as well.
